docs/user-tutorial/system-config.md

```diff
@@ -12,7 +14,21 @@ id: system-config
 - [Accelerator](#accelerator)
 - [PCIe](#pcie)
 
-## Parameter amd Details
+## Usage
+
+1. [Install SuperBench](../getting-started/installation.mdx) on the local machine.
+
+2. Prepare the host file of the tested GPU nodes using [Ansible Inventory](../getting-started/configuration.md#ansible-inventory) on the local machine.
+
+3. After installing the Superbnech and the host file is ready, you can start to collect the sys info automatically using  `sb node-info` command. The detailed command can be found from [SuperBench CLI](../cli.md).
+
+  ```
+  sb node-info -f host.ini --output-dir ${output-dir}
```

Comment:
currently you can only run "sb node-info" after "sb deploy" succeeds?

Reply:
no, updated 2 use cases